jupyterhub/values.yaml

```diff
@@ -18,10 +18,12 @@ hub:
   consecutiveFailureLimit: 5
   activeServerLimit:
   deploymentStrategy:
-    # sqlite-pvc backed hub requires Recreate strategy to work
+    ## type: Recreate
+    ## - a sqlite-pvc backed hubs require this as that storage can only be bound
+    ##   to one pod at the time.
+    ## - jupyterhub isn't designed to work with multiple replicas against the
+    ##   database
     type: Recreate
-    # This is required for upgrading to work
-    rollingUpdate:
```

Comment:
Did you remove this on purpose? I don't know why it is here really other than what the comment says but wanted to double check.

Reply:
Yepp, I asked for clarification about this comment before and it makes sense to remove this not only to me but the author of the comment. I actually forgot who that was but I did look it up before.